### PR TITLE
Patch up 2.5.2

### PR DESCRIPTION
**Pull request by JanneKiiskila**
To fix the build issues we are having with 2.5.1 release.

### CODE REVIEW

**Comment by petedyerarm** [comment]
We could also take a copy of the fixed version of runc-opencontainers_git.bb into recipes-containers/runc

**Comment by JanneKiiskila** [comment]
Good idea.

**Comment by petedyerarm** [comment]
Also https://github.com/PelionIoT/meta-pelion-edge/recipes-misc/pelion-version/files/BUILDMMU.txt need updating

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Pelion Edge 2.5.2
+
+- Fix compilation error related to `mbed-fcce`.
+- Pull in `opencontainer/runc` recipe to `meta-pelion-edge` to fix master to main rename build breakage.
+
```
Comment (petedyerarm) [inline CHANGELOG.md:5]: Think we've changed more than this.

**Review by petedyer11 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (JanneKiiskila) [reply CHANGELOG.md:8]: True, updated the CHANGELOG.md accordingly.

**Review by petedyer11 — APPROVED** [review]
LGTM

**Pull request merged**